sql/core/src/test/scala/org/apache/spark/sql/connector/SupportsCatalogOptionsSuite.scala

```diff
@@ -271,6 +275,47 @@ class SupportsCatalogOptionsSuite extends QueryTest with SharedSparkSession with
     }
   }
 
+  test("mock time travel test") {
+    sql(s"create table $catalogName.t123456789 (id bigint) using $format")
+    sql(s"create table $catalogName.t234567891 (id bigint) using $format")
+
+    val df1 = spark.range(10)
+    df1.write.format(format).option("name", "t123456789").option("catalog", catalogName)
+      .mode(SaveMode.Append).save()
+
+    val df2 = spark.range(10, 20)
+    df2.write.format(format).option("name", "t234567891").option("catalog", catalogName)
+      .mode(SaveMode.Overwrite).save()
+
+    // load with version
+    checkAnswer(load("t", Some(catalogName), Some("123456789")), df1.toDF())
```

Comment:
this is hard to read. Please name the parameter: `version = Some("123456789")`

Reply:
Sorry, fixed